tests/manual/test-procedure.rst

```diff
@@ -0,0 +1,282 @@
+Manual Test Procedure
+=====================
+
+Intro
+-----
+
+Until we have a comprehensive set end-to-end tests, we will have to carry
+some tests manually to ensure all the user-facing features of the Python Editor
+are working as expected.
+
+We need to be pragmatic about the amount of tests we carry manually, so this
+Test Procedure tries to balance coverage vs test duration.
+
+
+Nomenclature and Definitions
+----------------------------
+
+- **Python Editor**: This constitutes the whole web app.
+
+- **Text Editor**: The text or code editor (using the Ace framework) inside the
+  Python Editor.
+
+- **Load by editor drag&drop**: Load a file into the Python Editor by dragging
+  a file from the user OS, to the Text Editor area.
+
+- **Load by drop area drag&drop**: Load a file into the Python Editor by
+  clicking on the "Load button" and then dragging a file from the user OS to
+  the large grey box that says "Drag and drop a .hex or .py file in here to
+  open it".
+
+- **Load by file picker**: Load a file into the Python Editor by clicking on
+  the "Load button", then clicking on the "Or pick a file" link, then clicking
+  on the "Browse" button, selecting a file in the browser file picker, and
+  finally clicking on the "Load" button.
+
+- **Load by any method**: Load a file into the Python Editor using either
+  of the three options: "Load by editor drag&drop", "Load by drop area
+  drag&drop", or "Load by file picker".
+
+- **Connect to the REPL**: Use any type of serial terminal to connect with the
+  micro:bit, and send a Ctrl+C command to enter the REPL.
+
+- **Script Name**: The name of the script stored on the top right side of the
+  Python Editor. This can be often confused with a search box.
+
+- **Test Case**: A set of test steps to verify a related set of conditions.
+
+
+Preparation
+-----------
+
+Ensure you have everything listed in the "Prerequisites" section.
```

Comment:
Is this a section of this file or will there be a Prerequisites in each test file? Or is it the 'You will need' bit below? Perhaps clarify

Reply:
Good catch, that was the old name for the "You will need" section.